crates/astria-sequencer/src/state_ext.rs

```diff
@@ -41,6 +41,22 @@ pub(crate) trait StateReadExt: StateRead {
         Time::parse_from_rfc3339(&String::from_utf8_lossy(&bytes))
             .context("failed to parse timestamp from raw timestamp bytes")
     }
+
+    #[instrument(skip(self))]
+    async fn get_storage_version_by_height(&self, height: u64) -> Result<u64> {
+        let key: Vec<u8> = format!("storage_version/{height}").into();
+        let Some(bytes) = self
+            .nonconsensus_get_raw(key.as_slice())
```

Comment:
2 notes:

1. I tried to find this function in penumbra and it turns out they have already renamed it to [`nonverifiable_get_raw`](https://github.com/penumbra-zone/penumbra/commit/4eab19bc8defa1c98695e01d0b7a9f1f10b610a6). Should we bump our dependency?
2. Deref coercion should work here:
```suggestion
            .nonconsensus_get_raw(&key)
```

Reply:
yep I think we can bump our dep, will do in another PR

Reply:
ah turns out this wasn't in the v0.56 release lol